test-files/cases/attribute/unifying/highway-2927-bridge-1/Expected.osm

```diff
@@ -4284,7 +4284,52 @@
         <tag k="osm_id" v="-999999"/>
         <tag k="hoot:status" v="3"/>
         <tag k="error:circular" v="15"/>
-        <tag k="hoot:id" v="-7902"/>
+        <tag k="hoot:id" v="-8057"/>
+    </way>
+    <way visible="true" id="-408" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-26"/>
+        <nd ref="-513"/>
+        <tag k="source" v="Yahoo;osm:line"/>
+        <tag k="way_area" v="-999999"/>
+        <tag k="license" v="This data is made available under the Open Database License: http://opendatacommons.org/licenses/odbl/1.0/."/>
+        <tag k="oneway" v="yes"/>
+        <tag k="tags" v="\&quot;source:datetime\&quot;=&gt;\&quot;2018-10-30T13:05:51Z\\"/>
+        <tag k="name" v="Av. Sur 4"/>
+        <tag k="lanes" v="3"/>
+        <tag k="cycleway" v="no"/>
+        <tag k="surface" v="asphalt"/>
+        <tag k="sidewalk" v="both"/>
+        <tag k="maxspeed" v="40"/>
+        <tag k="attribution" v="osm"/>
+        <tag k="source:ingest:datetime" v="2019-01-17T19:50:05.667Z"/>
+        <tag k="z_order" v="-999999"/>
+        <tag k="highway" v="secondary"/>
+        <tag k="source:datetime" v="2015-08-24T16:13:18.000Z;2019-01-17T19:50:06Z"/>
+        <tag k="foot" v="yes"/>
+        <tag k="uuid" v="{337b5b66-e2e1-4cb7-83e6-bad10ea5d108}"/>
+        <tag k="bicycle" v="yes"/>
+        <tag k="osm_id" v="-999999"/>
+        <tag k="hoot:status" v="3"/>
+        <tag k="error:circular" v="15"/>
+        <tag k="hoot:id" v="-408"/>
+    </way>
+    <way visible="true" id="-407" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-39"/>
+        <nd ref="-38"/>
+        <tag k="source:ingest:datetime" v="2019-01-17T19:50:05.710Z"/>
+        <tag k="osm_id" v="-999999"/>
+        <tag k="license" v="This data is made available under the Open Database License: http://opendatacommons.org/licenses/odbl/1.0/."/>
+        <tag k="tags" v="\&quot;source:datetime\&quot;=&gt;\&quot;2018-10-30T13:05:51Z\\"/>
+        <tag k="z_order" v="-999999"/>
+        <tag k="uuid" v="{de291bb3-eeb7-46a5-9ccd-03eb6c52f288}"/>
+        <tag k="source" v="osm:line"/>
+        <tag k="way_area" v="-999999"/>
+        <tag k="attribution" v="osm"/>
+        <tag k="source:datetime" v="2019-01-17T19:50:06Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="error:circular" v="15"/>
+        <tag k="hoot:id" v="-407"/>
```

Comment:
This one is another bad split that would probably be remedied with the "dog-leg" improvements at the end of a way.